src/components/input/range/index.tsx

```diff
@@ -25,22 +39,23 @@ export class SmoothlyInputRange {
 
 	render() {
 		return (
-			<Host style={{ "--left-adjustment": `${(this.value / this.max) * 100}%` }}>
+			<Host style={{ "--left-adjustment": `${this.value ? (this.value / this.max) * 100 : 0}%` }}>
```

Comment:
As far as i can see this css variable is still unused. Is there more changes incoming that uses this? =]

Reply:
yes there are changes coming that will use this!